### ISSUE TITLE
OnPrem only?

### ISSUE THREAD

**Issue by j2associates** [MicrosoftDocs/dynamics365smb-devitpro-pb#1920]
The sample code includes the following line

FileTest.Open('c:\XMLDocs\NewTest.txt');  

If this holds true, there should be a Note somewhere near the top of the topic about it being On Prem only. Perhaps the topic could be changed to include a cloud based version also.

Thanks

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 04fb2aaa-8676-c0ae-434b-eb3d85bddca4
* Version Independent ID: 9d634195-090b-8b48-1059-e45233d7dcba
* Content: [Adding barcodes to reports - Business Central](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-report-add-barcodes)
* Content Source: [dev-itpro/developer/devenv-report-add-barcodes.md](https://github.com/MicrosoftDocs/dynamics365smb-devitpro-pb/blob/main/dev-itpro/developer/devenv-report-add-barcodes.md)
* Service: **dynamics365-business-central**
* GitHub Login: @jswymer
* Microsoft Alias: **jswymer**

**Comment by jswymer** [comment]
Hello, 

The linked article, [Adding barcodes to reports](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-report-add-barcodes), doesn't include the sample code that you mention. Could it be you're referring to the [InStream.ReadText(var Text [, Integer]) Method](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/methods-auto/instream/instream-readtext-method) article? Sorry if I've misunderstood something.

Thanks

**Comment by j2associates** [comment]
You are correct, the comment should refer to the InStream.ReadText(var Text [, Integer]) Method article. My apologies.
